src/hotspot/share/adlc/archDesc.cpp

```diff
@@ -24,7 +24,7 @@
 
 
 // archDesc.cpp - Internal format for architecture definition
-#include <unordered_set>
+#include <unordered_set> // do not reorder
```

Comment:
I *guess* that this has to do with the assert define. I think Kim has another workaround for that.